src/node/http.js

```diff
@@ -21,6 +22,16 @@ function deviceInformation(pkg) {
   }).join(' ');
 }
 
+function deviceInformation2(pkg) {
+  return {
+    hv: 1,
+    os: os.platform(),
+    ov: os.release(),
+    sdk: pkg.name,
+    pv: os.release()
```

Comment:
Why are the `OS Version` and the `Platform Version` the same value here?

Reply:
This was a mistake. I have fixed it.